low-level/files/opt/seagate/sspl/bin/sspl_setup

```diff
@@ -1,97 +1,234 @@
+        print(f"{self._script_dir}/{self.script}", self.name, ' '.join(self._args))
+        # Cmd._call_script(f"{self._script_dir}/{self.script}", self.name, self._args)
+
+
+class ConfigCmd(Cmd):
+    """Setup Config Cmd"""
+    name = "config"
+    script = "sspl_config"
+
+    def __init__(self, args):
+        super().__init__(args)
+
+    def process(self):
+        # TODO: Import relevant python script here for further execution.
+        print(f"{self._script_dir}/{self.script}", self.name, ' '.join(self._args))
+        # Cmd._call_script(f"{self._script_dir}/{self.script}", self.name, self._args)
+
+
+class TestCmd(Cmd):
+    """SSPL Test Cmd"""
+    name = "test"
+    script = "sspl_test"
+
+    def __init__(self, args):
+        super().__init__(args)
+
+    def process(self):
+        # TODO: Import relevant python script here for further execution.
+        print(f"{self._script_dir}/{self.script}", self.name, ' '.join(self._args))
+        # Cmd._call_script(f"{self._script_dir}/{self.script}", self.name, self._args)
+
+
+class SupportBundleCmd(Cmd):
+    """SSPL Support Bundle Cmd"""
+    name = "support_bundle"
+    script = "sspl_bundle_generate"
+
+    def __init__(self, args):
+        super().__init__(args)
+
+    def process(self):
+        # TODO: Import relevant python script here for further execution.
+        print(f"{self._script_dir}/{self.script}", self.name, ' '.join(self._args))
+        # Cmd._call_script(f"{self._script_dir}/{self.script}", self.name, self._args)
+
+
+class ManifestSupportBundleCmd(Cmd):
+    """Manifest Support Bundle Cmd"""
+    name = "manifest_support_bundle"
+    script = "manifest_support_bundle"
+
+    def __init__(self, args):
+        super().__init__(args)
+
+    def process(self):
+        # TODO: Import relevant python script here for further execution.
+        print(f"{self._script_dir}/{self.script}", self.name, ' '.join(self._args))
+        # Cmd._call_script(f"{self._script_dir}/{self.script}", self.name, self._args)
+
+
+class ResetCmd(Cmd):
+    """Setup Reset Cmd"""
+    name = "reset"
+    script = "sspl_reset"
+
+    def __init__(self, args):
+        super().__init__(args)
+
+    def process(self):
+        # TODO: Import relevant python script here for further execution.
+        print(f"{self._script_dir}/{self.script}", self.name, ' '.join(self._args))
+        # Cmd._call_script(f"{self._script_dir}/{self.script}", self.name, self._args)
+
+
+class CheckCmd(Cmd):
+    """SSPL Check Cmd"""
+    name = "check"
+    SSPL_CONFIGURED=f"/var/{PRODUCT_FAMILY}/sspl/sspl-configured"
+
+    def __init__(self, args):
+        super().__init__(args)
+
+    def process(self):
+        validate_consul_config.validate_config()
+        if os.path.exists(self.SSPL_CONFIGURED):
+            sys.exit(0)
+
+        logger_cmd = f"logger -i -p local3.err 'SSPL is not configured. Run provisioner scripts in {self._script_dir}.'"
```

Comment:
use syslog routine of python, we should not use this cmd line.

Reply:
Done